tsconfig.json

```diff
@@ -0,0 +1,35 @@
+{
+  "compilerOptions": {
+    "module": "commonjs",
+    "target": "es6",
```

Comment:
This could target es2023, which would avoid downconverting various modern features to something more compatible.

There is a preset you could use provided by `@companion-module/tools` instead of defining all of this yourself

Reply:
done, but add the dist outDir to not clutter the working dir